examples/Steel_Hall/Program.cs

```diff
@@ -0,0 +1,1291 @@
+                Console.WriteLine("Number of mesh nodes: " + mesh_Statistics.node_elements);
+                Console.WriteLine("Number of 1D elements: " + mesh_Statistics.member_1D_finite_elements);
+                Console.WriteLine("Number of surface element: " + mesh_Statistics.surface_2D_finite_elements);
+                Console.WriteLine("Number of volume elements: " + mesh_Statistics.solid_3D_finite_elements);
+                #endregion
+#endif
+                calculation_message[] calculationMessages = model.calculate_all(true);
+                if (calculationMessages.Length != 0)
+                {
+                }
+                else
+                {
+                    Console.WriteLine("Calculation finished successfully");
+                }
+
+                // printout result messages
+                #region Results
+                bool modelHasAnyResults = model.has_any_results();
+
+                if (modelHasAnyResults)
+                {
+                    Console.WriteLine("Model has results.");
+                }
+                else
+                {
+                    Console.WriteLine("Model has no results.");
+                }
+
+                bool modelHasLC2Calculated = model.has_results(case_object_types.E_OBJECT_TYPE_LOAD_CASE, liveLoad.no);
+                if (modelHasLC2Calculated)
+                {
+                    Console.WriteLine("Model has LC2 results.");
+                }
+                else
+                {
+                    Console.WriteLine("Model has no LC2 results.");
+                }
+
+                bool modelHasLC3Calculated = model.has_results(case_object_types.E_OBJECT_TYPE_LOAD_CASE, windX.no);
+                if (modelHasLC3Calculated)
+                {
+                    Console.WriteLine("Model has LC3 results.");
+                }
+                else
+                {
+                    Console.WriteLine("Model has no LC3 results.");
+                }
+
+                bool modelHasLC4Calculated = model.has_results(case_object_types.E_OBJECT_TYPE_LOAD_CASE, windY.no);
+                if (modelHasLC4Calculated)
+                {
+                    Console.WriteLine("Model has LC4 results.");
+                }
+                else
+                {
+                    Console.WriteLine("Model has no LC4 results.");
+                }
+
+                bool modelHasLC5Calculated = model.has_results(case_object_types.E_OBJECT_TYPE_LOAD_CASE, snow.no);
+                if (modelHasLC5Calculated)
+                {
+                    Console.WriteLine("Model has LC5 results.");
+                }
+                else
+                {
+                    Console.WriteLine("Model has no LC5 results.");
+                }
+                #endregion
+
+                #region export results
+                model.export_result_tables_with_detailed_members_results_to_csv("C:\\Users\\GoebelR\\Documents\\Webservices\\testmodels\\CSV");
```

Comment:
Don't add absolute path use e.g. 
```suggestion
                string currentDir = Directory.GetCurrentDirectory();
                model.export_result_tables_with_detailed_members_results_to_csv(currentDir + @"\CSV\");
```